cert_issuer/merkle_tree_generator.py

```diff
@@ -86,7 +86,7 @@ def get_proof_generator(self, tx_id, verification_method, chain=Chain.bitcoin_ma
 
 
 def to_source_id(txid, chain):
-    if chain == Chain.bitcoin_mainnet or Chain.bitcoin_testnet or Chain.ethereum_mainnet or Chain.ethereum_ropsten:
```

Comment:
Before this change, the condition was always true.
That looked bug to me. So, could you review that this change correct is?

Reply:
No, I haven't found any issue. I just realized this condition was always `true`, it looks wrong.

I'm not sure what the correct condition is, so I'd like to follow your idea. I can revert or change what you want.

Reply:
it's not true if the chain is mocknet, which is the intent. Your change is good.